packages/psss-server/examples.http

```diff
@@ -24,22 +24,31 @@ content-type: {{contentType}}
 
 ### /test
 GET http://{{host}}/{{version}}/test HTTP/2.0
-Cookie: {{sessionCookie}}
+
+### Fetch Alerts
+GET http://{{host}}/{{version}}/alerts/active?orgUnitCodes=TO,FJ&startWeek=2016W07&endWeek=2021W20 HTTP/2.0
 
 ### Fetch Countries
 GET http://{{host}}/{{version}}/country HTTP/2.0
 
 ### Fetch Country Sites
 GET http://{{host}}/{{version}}/country/TO/sites HTTP/2.0
 
-### Fetch Country Weekly Report Data
+### Fetch Weekly Report Data (single country)
 GET http://{{host}}/{{version}}/weeklyReport/TO?startWeek=2016W07&endWeek=2021W07 HTTP/2.0
-Cookie: {{sessionCookie}}
 
-### Save Country Weekly Report Data
+### Fetch Weekly Report Data (single country, single syndrome)
+GET http://{{host}}/{{version}}/weeklyReport/TO?startWeek=2021W16&endWeek=2021W16&syndrome=ILI HTTP/2.0
+
+### Fetch Weekly Report Data (all sites in a country)
+GET http://{{host}}/{{version}}/weeklyReport/TO/sites?startWeek=2021W18&endWeek=2021W18 HTTP/2.0
+
+### Fetch Weekly Report Data (all sites in a country, single syndrome)
+GET http://{{host}}/{{version}}/weeklyReport/TO/sites?startWeek=2021W18&endWeek=2021W18&syndrome=ILI HTTP/2.0
+
+### Save Weekly Report Data (single country)
```

Comment:
Really appreciate you updating this file and giving clear descriptions for the endpoints 🙏